packages/core/src/schema/plugins/fragments/platformBase.ts

```diff
@@ -3,6 +3,7 @@ import { z } from 'zod';
 // DEPRECATED?
 const Enabled = z.boolean().default(true).describe('Marks plugin platform enabled or disabled');
 const Disabled = z.boolean().default(false).describe('Marks plugin platform disabled');
+const ForceLinking = z.boolean().default(false).describe('Marks plugin as not able to be autolinked');
```

Comment:
please add a more detailed description, something like "Packages that cannot be autolinked yet can still be added to MainApplication PackageList dynamically by setting this to true"